core/interop/IDynamicInterfaceCastable/src/ManagedApp/Program.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+
+namespace IDynamicInterfaceCastableSample
+{
+    [Guid("0412BF07-0261-4191-B5DA-9B86340931CB")]
+    public interface IGreet
+    {
+        void Hello();
+    }
+
+    [Guid("B88C195F-3052-467A-97D2-817A1698AAFB")]
+    public interface ICompute
+    {
+        int Sum(int a, int b);
+    }
+
+    class Program
+    {
+        [DynamicInterfaceCastableImplementation]
+        private interface IGreetImpl : IGreet
+        {
+            [StructLayout(LayoutKind.Sequential)]
+            private struct IGreetVtbl
+            {
+                public IntPtr QueryInterface;
+                public IntPtr AddRef;
+                public IntPtr Release;
+                public _Hello Hello;
+            }
+
+            private delegate void _Hello(IntPtr _this);
```

Comment:
```suggestion
            private delegate void Hello(IntPtr _this);
```

I do not believe I've ever seen delegates name with a leading `_` and capital letter. I prefer this style instead, no leading `_`.

Reply:
Removed the `_` and added `Delegate` as a suffix (to clearly distinguish from the interface method / struct property) for all the delegate names.